components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/bindings/impl/AbstractTokenBinder.java

```diff
@@ -43,6 +44,8 @@
  */
 public abstract class AbstractTokenBinder implements TokenBinder {
 
+    private static final String COMMONAUTH_COOKIE = "commonAuthId";
```

Comment:
There should be a constant defined for this in the identity-framework. Better to reuse that.